app/controllers/languages_controller.rb

```diff
@@ -0,0 +1,66 @@
+class LanguagesController < ApplicationController
+  before_action :set_language, only: %i[edit update add_to_volunteer remove_from_volunteer]
+
+  def new
+    authorize Language
+    @language = Language.new
+  end
+
+  def edit
+    authorize @language
+  end
+
+  def create
+    authorize Language
+    @language = Language.new(language_params)
```

Comment:
style: these two lines can be merged- `Language.new(language_params.merge(casa_org: current_organization))`

Reply:
absolutely!